src/main/kotlin/no/nav/hjelpemidler/soknad/db/domain/SøknadForBruker.kt

```diff
@@ -454,7 +455,15 @@ private fun sengeInfo(hjelpemiddel: JsonNode): SengeInfo? {
             "HarFraFor" -> MadrassValg.HarFraFor
             null -> null
             else -> throw RuntimeException("Ugyldig sitteputeValg")
-        }
+        },
+        høyGrindValg = if (høyGrindValg != null) {
```

Comment:
Her trur eg vi kan forenkle med treeToValue. Tilsvarande som for oppreisingstol (linje 308):
```
private fun oppreisningsStolInfo(hjelpemiddel: JsonNode): OppreisningsStolInfo? {
    val oppreisningsStolInfo = hjelpemiddel["oppreisningsStolInfo"] ?: return null
    return objectMapper.treeToValue<OppreisningsStolInfo>(oppreisningsStolInfo)
}
```

Så noko sånnt som dette:
```
høyGrindValg = høyGrindValg?.let{ objectMapper.treeToValue<HøyGrindValg>(it) }
```